.buckconfig

```diff
@@ -0,0 +1,6 @@
+
+[android]
+  target = Google Inc.:Google APIs:23
```

Comment:
Is there any reason for using API 23? Because it only cover 4.3% of users ( according to android studio).